WORKSPACE

```diff
@@ -30,6 +30,39 @@ load("@io_bazel_rules_bison//bison:bison.bzl", "bison_register_toolchains")
 
 bison_register_toolchains()
 
+load("//third_party/gems:rules.bzl", "gemfile_lock_deps")
+
+gemfile_lock_deps(
+    name = "gems",
+
+    # TODO: figure out how to discover these labels during the loading phase,
+    # rather that listing them out explicitly.
+    gemfile_locks = [
+        "//gems/sorbet/test/snapshot:{}/src/Gemfile.lock".format(test)
+        for test in [
+            "partial/bad-hash",
+            "partial/bad-t",
+            "partial/codecov",
+            "partial/create-config",
+            "partial/explosive-object",
+            "partial/fake-object",
+            "partial/fake-rails",
+            # "partial/local_gem",
```

Comment:
What's up with this guy: local_gem?

Reply:
The local gem causes the build to fail, as the gem does not exist on rubygems.org. I think that it's possible to handle this case better, but I wanted to get this work merged so that it didn't continue to sit on a branch. I'll add a comment indicating what I think needs to be done to support this.

Reply:
I've fixed this by only looking for gem names in the `GEM` section.